apps/mobile/app/screens/settings/settings-data.tsx

```diff
@@ -89,8 +89,8 @@ export const settingsGroups: SettingSection[] = [
           return isBasic || !user.subscription?.type
             ? "Subscribe to Pro"
             : isTrial
-            ? "Your free trial has started"
-            : "Subscription details";
+              ? "Your free trial has started"
+              : "Subscription details";
```

Comment:
Sorry about this edit. I'm fighting with my editor. It keeps on auto-formatting this code.